lib/plugin.js

```diff
@@ -30,7 +30,7 @@ function plugin (pluginOptions) {
     }
 
     try {
-      if (extractSignature(req) === constructSignatureString(req, options)) {
+      if (options.extractSignature(req) === options.constructSignatureString(req, options)) {
```

Comment:
I think these should be unpacked first
```suggestion
      const {
        extractSignature,
        constructSignatureString
      } = options
      if (extractSignature(req) === constructSignatureString(req, options)) {
```

Reply:
this may be covered by standardjs